monkey/monkey_island/cc/setup/data_dir.py

```diff
@@ -90,3 +92,23 @@ def _data_dir_version_mismatch_exists(data_dir_path: Path) -> bool:
     island_version = get_version()
 
     return island_version != data_dir_version
+
+
+def _copy_plugins_into_data_dir(data_dir_path: Path):
+    plugin_path = Path(MONKEY_ISLAND_ABS_PATH) / "plugins"
+    try:
+        plugins_destination_path = _create_plugins_dir(data_dir_path)
+
+        plugin_tar_paths = list(plugin_path.glob("*.tar"))
+        for plugin_tar_path in plugin_tar_paths:
+            shutil.copy2(plugin_tar_path, plugins_destination_path)
+    except FileNotFoundError:
+        logger.exception(f"An error occured while copying plugins to data_dir: {data_dir_path}")
```

Comment:
Put the try/except inside the loop, around the copy statement. The way it is right now, if one file can't be copied for some reason, it'll catch the error, log, and exit the function. It won't attempt the ones next in line.